staging/src/k8s.io/apiextensions-apiserver/pkg/features/kube_features.go

```diff
@@ -30,27 +32,39 @@ const (
 
 	// owner: @alexzielenski
 	// alpha: v1.28
+	// beta: v1.30
 	//
 	// Ignores errors raised on unchanged fields of Custom Resources
 	// across UPDATE/PATCH requests.
 	CRDValidationRatcheting featuregate.Feature = "CRDValidationRatcheting"
 
 	// owner: @jpbetz
 	// alpha: v1.30
+	// beta: v1.31
+	// ga: v1.32
```

Comment:
nit: not a problem since you have a promotion PR after, but comment should go with the promotion PR

Reply:
oops, good catch